etc/metrics_metadata.json

```diff
@@ -803,6 +863,18 @@
         "notes": "The larger the amount of data being buffered to be sent will cause the goxdcr process to take up more memory.  If too much memory is being used, consider decreasing the number of nozzles or tune such that less data will be buffered",
         "stability": "committed"
     },
+    "xdcr_source_sync_xattr_removed": {
+        "type": "counter",
+        "help": "Number of mutations with source mobile XATTR removed",
```

Comment:
Replace XATTR by its full technical name "extended attributes" for purpose of outward facing documentation

Reply:
Done